server/actions/events.js

```diff
@@ -16,15 +16,29 @@ export async function createEvent(newEventData, next) {
   });
 }
 
-export async function getEvents(next) {
+export async function getEvents(startDate, endDate, next) {
 
   await dbConnect();
 
-  return EventData.find({})
-  .then((events) => {
-    return events;
-  })
-  .catch(next);
+  if (!startDate && !endDate){
```

Comment:
Could we also additionally add a check where only one of startDate or endDate is provided? For example: if only start date is provided return all events after that date and vice versa. This would help requests to display past events and future events.